pkg/trace/pb/remote_rates.go

```diff
@@ -0,0 +1,408 @@
+package pb
```

Comment:
Let's suffix the name of this file with `_gen.go`. All the other msgpack files are like that. It will make it clear on an overview.

Also, I think you forgot to commit the `.proto` file?